sr_multi_moveit/sr_moveit_planner_benchmarking/launch/benchmarking.launch

```diff
@@ -1,6 +1,6 @@
 <launch>
   <arg name="planning" default="ompl"/>
-  <arg name="visualization" default="False"/>
+  <arg name="visualization" default="True"/>
```

Comment:
why have you turned on visualization? I guess for Jenkins it should be off by default?

Reply:
True, I turned back off